internal/cli/atlas/deployments/options/deployment_opts_remove.go

```diff
@@ -16,16 +16,27 @@ package options
 import "context"
 
 func (opts *DeploymentOpts) Remove(ctx context.Context) error {
-	if _, err := opts.PodmanClient.RemoveContainers(ctx, opts.LocalMongodHostname(), opts.LocalMongotHostname()); err != nil {
-		return err
+	volumes := []string{opts.LocalMongodDataVolume(), opts.LocalMongotDataVolume(), opts.LocalMongoMetricsVolume()}
```

Comment:
isn't  `opts.LocalMongodDataVolume()` found by container inspect 2 lines below?
If yes, maybe this collection should include only the last 2 volumes (used by mongot)

Reply:
so what if the container is not present? then we would miss it

Reply:
is podman ok with duplicate names for the delete command (`[non-blocking if podman is ok with duplicate volume names]`)